src/main/resources/noUser/adminview.html

```diff
@@ -6,6 +6,26 @@
     <meta charset="UTF-8">
     <title>Admin</title>
     <link href="css/admin.css" rel="stylesheet">
+    <style>
+        .userTable {
+    padding-left: 5%;
+    padding-right: 5%;
+}
+
+.admin-button{
+    padding: 0.4rem 0.4em;
+    font-size: 1rem;
+    line-height: 0.7;
+    border-radius: 0.3rem;
+    color: white;
+    background-color: #007bff;
+    border-color: #007bff;
+}
+.admin-table{
+    background-color: #d0deeb;
+}
+
+    </style>

```

Comment:
should this be in a css file?

Reply:
We could make a change to the Spring Security Config files to allow css files to work, but as it stands right now, this is the simplest way to style

Reply:
Fixed issue, I was referencing wrong stylesheet